crates/libcontainer/src/syscall/linux.rs

```diff
@@ -559,6 +559,24 @@ impl Syscall for LinuxSyscall {
         }?;
         Ok(())
     }
+
+    fn set_io_priority(&self, class: i64, priority: i64) -> Result<()> {
+        let ioprio_who_pgrp = 1;
+        let iop = (class << 13) | priority;
+        match unsafe {
+            libc::syscall(
+                libc::SYS_ioprio_set,
+                ioprio_who_pgrp as libc::c_int,
```

Comment:
Can we move the type information to the `let` above so we don't have to cast?

Reply:
update